services/QuillLMS/client/app/bundles/Teacher/components/content_hubs/subject_page_teacher_center_articles.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import ReactMarkdown from 'react-markdown';
+
+import PreviewCard from '../shared/preview_card.jsx';
+import { requestGet } from '../../../../modules/request';
+
+const GETTING_STARTED_COLLECTION_ID = 477
+const INTEGRATING_QUILL_EVIDENCE_ARTICLE_ID = 666
+
+const TeacherCenterArticleCard = ({ blogPostId, }) => {
+  const [blogPost, setBlogPost] = React.useState(null);
+  const [blogPostAuthor, setBlogPostAuthor] = React.useState(null);
+
+  React.useEffect(() => {
+    getBlogPost(blogPostId)
+  }, [])
+
+  function getBlogPost(id: string) {
+    requestGet(`/featured_blog_post/${id}`,
+      (data) => {
+        if(data.blog_post) {
+          const { blog_post, author } = data;
+          setBlogPost(blog_post)
+          setBlogPostAuthor(author)
+        }
+      }
+    )
+  }
+
+  function renderPreviewContent({ title, slug, preview_card_content, external_link, footer_content }) {
+    if (footer_content) {
+      return(
+        <section className="content-section">
+          <h4>{title}</h4>
+          <ReactMarkdown className="preview-card" source={footer_content} />
+          <section className="footer-section">
+            {blogPostAuthor && <p className="author">{`By ${blogPostAuthor}`}</p>}
+            <a className="quill-button focus-on-light contained teal extra-small" href={`/teacher-center/${slug}`} rel="noopener noreferrer" target="_blank">Read</a>
+          </section>
+        </section>
+      )
+    }
+
+    return(
+      <PreviewCard
+        color="teal"
+        content={preview_card_content}
+        externalLink={!!external_link}
+        key={title}
+        link={external_link ? external_link : `/teacher-center/${slug}`}
+      />
+    )
+  }
+
+  if(!blogPost) { return <span /> }
+
+  return renderPreviewContent(blogPost)
+}
+
+const SubjectPageTeacherCenterArticles = () => (
+  <section className="teacher-center-articles">
+    <div className="container">
+      <h2>Learn more with these Teacher Center articles</h2>
+      <div className="articles-wrapper">
+        <TeacherCenterArticleCard blogPostId={GETTING_STARTED_COLLECTION_ID} />
+        <TeacherCenterArticleCard blogPostId={INTEGRATING_QUILL_EVIDENCE_ARTICLE_ID} />
```

Comment:
This is purely stylistic so take or leave it, but maybe it'd be cleaner to expand this if we used some sort of `map` call to build each of these `TeacherCenterArticleCard` instances rather than just copy/pasting the line with new constants in each of them.  Given that there are just two of them, this probably doesn't matter here, I'm just imagining a future where we had three or four more, which may not be likely.

Reply:
I agree that having just the two makes me feel like it's okay to leave for now, but I will definitely keep this in mind if we expand later (or start using different articles for different pages).